chapters/flow/index.md

```diff
@@ -1,4 +1,4 @@
-## Compose Your Workflow with the Flow API
+## Composing a Search Workflow with the Flow API
```

Comment:
This should be sentence case

Reply:
This is more of a case of the heading being the wrong level. It should be `#`, not `##`, so case should remain as title. I'll fix the heading level bug though